tgui/packages/tgui/interfaces/pda/pda_messenger.js

```diff
@@ -153,14 +153,12 @@ export const MessengerList = (props, context) => {
             <Button icon="bell" onClick={() => act('Ringtone')}>
               Set Custom Ringtone
             </Button>
-            <Button>
-              <Dropdown
-                selected={ringtone}
-                width="100px"
-                options={Object.keys(ringtone_list)}
-                onSelected={(value) => act('Available_Ringtones', { selected_ringtone: value })}
-              />
-            </Button>
```

Comment:
KEKW

Reply:
what the fuck dropdown was doing in button